make/Makefile.build.mk

```diff
@@ -25,6 +25,12 @@ build: go-check
 	${GO_BUILD_ENVVARS} ${GO} build \
 		-o ${GOPATH}/bin/kiali -ldflags "-X main.version=${VERSION} -X main.commitHash=${COMMIT_HASH}"
 
+build-linux:
```

Comment:
What is the use-case for this target? How are people to use it? Note that all non-hidden targets (those not prefixed with a period `.`) have a `##` comment that is output when you do a `make help`... so add a `##` comment here and explain briefly this target and how it is to be used. I assume it should be something like this:

```suggestion
## Allows you to perform a multi-arch build. Set TARGET_ARCH env var to a value containing space-separated names of architectures you want built (e.g. TARGET_ARCH="amd64 arm64").
build-linux:
```